hooks.ts

```diff
@@ -1,10 +1,12 @@
-const fs = require('fs');
-const path = require('path');
+import fs = require('fs');
+import path = require('path');
 
-module.exports = (app) => {
+import {Hook, registerHook} from 'openfin-service-tooling/utils/allowHook';
+
+registerHook(Hook.TEST_MIDDLEWARE, (app) => {
     // Sneakily return the test directory instead of the default one
     app.get('/provider/sample-app-directory.json', (req, res) => {
-        const testDirectory = JSON.parse(fs.readFileSync(path.join('res', 'test', 'sample-app-directory.json')));
+        const testDirectory = JSON.parse(fs.readFileSync(path.join('res', 'test', 'sample-app-directory.json')).toString());
```

Comment:
Why is this `toString` here? Wouldn't that just return `[Object object]` or something like that?

Reply:
Without it I get "Argument of type 'Buffer' is not assignable to parameter of type 'string'.". Will double check it's doing what I assume

Reply:
Yep, does what I expected. Which makes it werid that it worked in the first place. I think JSON.parse must specifically check for buffer objects and handle appropriately, but that's not reflected in the types

Reply:
Hmm, fair enough.